test/lower-case_spec.js

```diff
@@ -9,22 +9,29 @@ describe('lower-case Node', function () {
   });
 
   it('should be loaded', function (done) {
-    var flow = [{ id: "n1", type: "lower-case", name: "lower-case" }];
+
+    // Exported flow pasted as JSON string
+    var flow = '[{"id":"3912a37a.c3818c","type":"lower-case","z":"e316ac4b.c85a2","name":"lower-case","x":240,"y":320,"wires":[[]]}]';
```

Comment:
not sure why this is needed as above.  Just drop single quotes and it's equivalent to the original?  The original is easier to understand imho.

Reply:
I was using this as a test case for testing the export from the editor with debug nodes - that is why the id's are like that.

Reply:
ok.  how about an additional test called 'it should be loaded in exported flow'?

Reply:
No, no - I was using lower-case as a test bed.  The Id's and unused wires are what is exported from the editor.